<?xml version="1.0"?>
<launch>
    <!-- <rosparam command="load" file="$(find indoors_pbm)/config/thresholds/thresholds.yaml" /> -->
    <rosparam command="load" file="$(find indoors_pbm)/config/PBM.yaml" />
    <node pkg="indoors_pbm" type="indoors_pbm_node" name="PBM" output="screen"/>

</launch>

